staging/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Name: staging/values.yaml
+# Purpose: Provide Helm values customized for the Staging Jenkins env
+# Description: Overrides Jenkins image, JVM opts, and JCasC env variables
+
+controller:
+  image:
+    # Using the official public Jenkins LTS image as a placeholder.
+    repository: "jenkins/jenkins"
+    tag: "lts-jdk17"
+    pullPolicy: "IfNotPresent"
+
+# Defining resource requests and limits for the staging environment.
+resources:
+  requests:
+    cpu: "1000m"
+    memory: "2Gi"
+  limits:
+    cpu: "2000m"
+    memory: "4Gi"
+
+jenkinsJavaOpts: "-Xms1g -Xmx2g"
+
+jenkinsEnvVars:
+  JCASC_SYSTEM_MESSAGE: "Welcome to Jenkins (Staging Environment)"
+  JCASC_LOCATION_URL: ""
+  JCASC_LOCATION_ADMINADDRESS: "staging-admin@example.com"
```

Comment:
Do we want to put a real email address here so that the right folks get contacted?